README.md

```diff
@@ -0,0 +1,305 @@
+
+## Purpose
+
+This repository contains a sample Alexa skill that allows users to connect to
+Open edX. With this skill, users can access course information, check grades,
+and perform other Open edX-related tasks using voice commands with their
+Alexa-enabled devices.
+
+## The Sample Skill
+
+### Description
+
+The sample skill included in this repository is a basic demonstration of how to
+integrate Alexa with Open edX. It showcases how to handle user requests,
+authenticate with Open edX, and retrieve course data. Developers can use this
+as a starting point to build more advanced interactions for Open edX.
+
+### Features
+
+The sample skill includes the following features:
+
+- **Voice profile support**: The skill can identify users by their voice and
+  retrieve their course information. If the user is not recognized, the skill
+  will return a message indicating that the user is not recognized and the
+  interaction will finish.
+- **Internationalization support**: The skill is available in English and
+  Spanish. Developers can add support for other languages by adding the
+  appropriate translations.
+- **Custom Email Authentication**: The skill uses a custom authentication
+  mechanism to authenticate users with Open edX. By default, the skill uses the
+  email address of the Alexa account to authenticate with Open edX. The developers
+  can create a custom authentication mechanism to authenticate users with Open edX.
+
+### Usage
+
+To use this Alexa skill, you can use the Alexa simulator in the Amazon
+Developer Console or an Alexa-enabled device. The skill is available in English
+and Spanish. The current version of the skill supports the following
+interactions:
+
+- **Launch the skill**: "Alexa, open educational assistant."
+
+  > _Alexa Response_: "Welcome \<first_name\>, this is the Open edX assistant,
+  > I can provide you information about student metrics and important aspects
+  > of a course."
+
+  e.g:
+
+  > "_Alexa Response_: Welcome **John**, this is the Open edX assistant, I can
+  > provide you information about student metrics and important aspects of a
+  > course."
+
+- **Get course progress**: "Alexa, give me my progress in the course of
+  \<course-name\>."
+
+  > _Alexa Response_: "The progress for the student with
+  > username \<username\> in the course of \<coursename\> is \<percentage\>.
+
+  e.g:
+
+  > "_Alexa Response_: The progress for the student with username **johndoe**
+  > in the course of **Introduction to Linux** is **50%**."
+
+## Creating the Skill
+
+### Prerequisites
+
+Before setting up and running this sample skill, ensure you have the following
+prerequisites:
+
+1. An [Amazon Developer account](https://developer.amazon.com/) for creating
+   and configuring the skill.
+2. Installed [Node.js and npm](https://nodejs.org/en), [Python 3.8](https://www.python.org/downloads/),
+   and [git](https://git-scm.com/).
+3. Access to an Open edX platform with admin privileges.
```

Comment:
Why do I need admin privileges?

Reply:
In order to create the necessary application in the Django OAuth Toolkit

Reply:
admin refers to django staff or superuser?

Reply:
What about an user without staff/superuser permisions but with eox core API permissions? 
![image](https://github.com/eduNEXT/openedx-alexa-adaptor-template/assets/64440265/59abec4b-b962-4dd4-a552-43469111c47c)

Reply:
In this case, it would work for the eox-core API, but not for the edX API, to get the enrollments we need to have staff permissions.

Reply:
Good to know! Thanks